src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -1303,6 +1303,40 @@ private bool CanAddToGroup(object parameters)
             return false;
         }
 
+        private void SelectUpstream(object parameters)
+        {
+            NodeModel.SelectUpStreamNeighbours();
+        }
+
+        private bool CanSelectUpstream(object parameters)
+        {
+            //Add checks for can stream
+            return true;
+        }
+
+        private void SelectDownstream(object parameters)
+        {
+            NodeModel.SelectDownStreamNeighbours();
+        }
+
+        private bool CanSelectDownstream(object parameters)
+        {
+            //Add checks for can stream
+            return true;
+        }
+
+        private void SelectDownAndUpstream(object parameters)
+        {
+            NodeModel.SelectUpStreamNeighbours();
+            NodeModel.SelectDownStreamNeighbours();
+        }
+
+        private bool CanSelectDownAndUpstream(object parameters)
```

Comment:
Does Dynamo force that parameters argument? we dont really need it in any of these methods do we?

Reply:
The `DelegateCommand` constructor takes an `Action<object>` so I think this is needed, if not it throws an error. That is why this is used in other commands, regardless of not if they use the object.